reviews/tests.py

```diff
@@ -258,18 +258,24 @@ def test_auto_review(self):
         reasons = auto_review(self.proposal)
         self.assertEqual(len(reasons), 5)
 
-        self.study.stressful = YesNoDoubt.YES
+        self.study.risk = YesNoDoubt.YES
         self.study.save()
 
         reasons = auto_review(self.proposal)
         self.assertEqual(len(reasons), 6)
 
-        self.study.risk = YesNoDoubt.YES
+        self.study.proposal.researcher_risk = YesNoDoubt.YES
         self.study.save()
```

Comment:
The tests seem to pass fine without saving the object. But if you could save the proposal object here instead of the study at least it will be internally consistent, should there be a case (in the future) where saving does matter.